setup.py

```diff
@@ -38,7 +38,8 @@
         # nedded because users vocabulary was added here
         'plone.app.vocabularies>=2.1.12dev',
         # needed for compatibility with jQuery 1.9+
-        'Products.CMFPlone>=4.3.4'
+        'Products.CMFPlone>=4.3.4',
+        'plone.app.registry',
```

Comment:
we also need plone.api here. in plone 5, this is even a core  dependency, although it isn't used in the core